docs/infrastructure/hosted-server.rst

```diff
@@ -0,0 +1,102 @@
+RITlug Server
+=============
+
+RITlug has a hosted server, located in the Institute Hall Datacenter. It's currently owned and maintained by `Solomon Rubin (Serubin) <https://github.com/serubin/>`, however it periminently on lend to RITlug. He remains the primary contact for any major issues. RITlug related use will be maintained by disignated Club Server Admins. 
```

Comment:
Spelling errors. Possible interpretation: "...however, it is permanently on lend..."

"...will be maintained by designated Club Server Admins."

Reply:
Maybe "indefinitely" would be better than "permanently"?

Reply:
@Serubin Did you see the feedback here from @Tjzabel / @ct-martin?